data/migrations/20190515111950_LinkTable.js

```diff
@@ -0,0 +1,27 @@
+
+exports.up = function(knex, Promise) {
+return knex.schema.createTable('links', function(links){
+links.increments();
+links.integer('event_id');//declaring a foreign key
+links.foreign('event_id')//using the foreign key
+    .references('id')
+    .inTable('events');
+
+
+links.integer('user_id');//declaring the other foreign key
+links.foreign('user_id')//using this foreign key
```

Comment:
Line 6 is not necessary. The integer user_id is referencing the id from the users table.